packages/shared/src/components/filters/FollowingFilter.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { type ReactElement } from 'react';
+import Link from 'next/link';
+import { useAuthContext } from '../../contexts/AuthContext';
+import UserList from '../profile/UserList';
+import { useFollowingQuery } from '../../hooks/contentPreference/useFollowingQuery';
+import { ContentPreferenceType } from '../../graphql/contentPreference';
+import { FlexCentered } from '../utilities';
+import { checkFetchMore } from '../containers/InfiniteScrolling';
+import { AddUserIcon } from '../icons';
+import { IconSize } from '../Icon';
+import { useLazyModal } from '../../hooks/useLazyModal';
+
+export const FollowingFilter = (): ReactElement => {
+  const { user } = useAuthContext();
+  const { closeModal } = useLazyModal();
+  const queryResult = useFollowingQuery({
+    id: user.id,
+    entity: ContentPreferenceType.User,
+  });
+  const { data, isFetchingNextPage, fetchNextPage } = queryResult;
+  const users = data?.pages.reduce((acc, p) => {
+    p?.edges.forEach(({ node }) => {
+      acc.push(node.referenceUser);
+    });
+
+    return acc;
+  }, []);
+
+  return (
+    <UserList
+      users={users}
+      emptyPlaceholder={
+        <FlexCentered className="flex-col gap-4 px-6 py-10 text-center text-text-tertiary typo-callout">
+          <AddUserIcon size={IconSize.XXXLarge} />
+          <p>
+            You haven&apos;t follow any User yet.
+            <br /> Explore our{' '}
+            <Link
+              href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}users`}
+              passHref
+              onClick={() => {
+                closeModal();
+              }}
```

Comment:
Why do we need to manually close it?
Isn't just putting a `<a>` inside enough to force page load and close it?

Reply:
An anchor does indeed force page load, but it loads the _entire_ page again, not just next.js magic loading the new content

Reply:
Removed. Do note in preview it does an entire new page load because of it pointing to prod domain 😬